src/components/common/Select.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC, useState } from 'react';
+
+import {
+  MenuToggle,
+  MenuToggleStatus,
+  Select as PfSelect,
```

Comment:
why do u change it to Pfsleect, but SelectList is just SelectList?

Reply:
I mean, if you want to create a generic Select for our project isn't it better naming it like MTVSelect so we aware its our select and probably also generic/shared as it start with MTV? - just something to think about, its not a must

Reply:
To answer your first question, it's just a way for the names to not conflict in this file.

In regards to your 2nd comment, I'm open to that. I would want to stick with PascalCase. I would only want to do this for situations like this though, where what we're naming here conflicts with a PF component name.

Reply:
Updated the naming.